### DISCUSSION TITLE
Needed a blog for developers

### DISCUSSION THREAD

**Discussion by yousufkalim** [community/community#41805] [Other Feature Feedback, Questions, & Ideas]
### Product Feedback
Github is getting best and best, It would be great if you add the full fledged blog for the developers so where we can find the articles related to the development only.

**Comment by mohamedzhioua** [comment]
@yousufkalim  i agreee

**Comment by jfullstackdev** [comment]
what do you mean by that ? GitHub has already the Blog of its platform

as of individual, you can use GitHub Pages or GitHub Gist to do that because you already have followers that will see those, you can even do a forum if you want through GitHub Discussions

**Reply by yousufkalim** [reply]
As I said full-fledged blog with articles (e.g, dev.to, medium, the better programmer). Gist is for code sharing and pages for static website hosting. We cannot find articles from different authors at one place.
